frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -0,0 +1,300 @@
+  return {
+    ...defaultPidPinFormValues,
+    ...defaultLandValues,
+    ...defaultInformationFormValues,
+    latitude: '',
+    longitude: '',
+    address: defaultAddressValues,
+    buildings: [],
+    financials: defaultFinancials,
+  };
+};
+
+/**
+ * Do an in place conversion of all values to their expected API equivalents (eg. '' => undefined)
+ * @param values the parcel value to convert.
+ */
+export const valuesToApiFormat = (values: ISteppedFormValues<IFormParcel>): IFormParcel => {
+  values.data.pin = values?.data.pin ? values.data.pin : undefined;
+  values.data.pid = values?.data.pid ? values.data.pid : undefined;
+  const seperatedFinancials = (_.flatten(
+    values.data.financials?.map((financial: IFinancialYear) => _.values(financial)),
+  ) ?? []) as IFinancial[];
+  const allFinancials = filterEmptyFinancials(seperatedFinancials);
+
+  values.data.evaluations = _.filter(allFinancials, financial =>
+    Object.keys(EvaluationKeys).includes(financial.key),
+  );
+  values.data.fiscals = _.filter(allFinancials, financial =>
+    Object.keys(FiscalKeys).includes(financial.key),
+  );
+  values.data.landArea = +values.data.landArea;
+  values.data.financials = [];
+  return values.data;
+};
+
+interface ILandSubformProps {
+  handlePidChange: (pid: string) => void;
+  handlePinChange: (pin: string) => void;
+  setMovingPinNameSpace: (nameSpace: string) => void;
+  handleGeocoderChanges: any;
+}
+
+const Form: React.FC<ILandSubformProps> = ({
+  handleGeocoderChanges,
+  setMovingPinNameSpace,
+  handlePidChange,
+  handlePinChange,
+}) => {
+  // access the stepper to later split the form into segments
+  const stepper = useFormStepper();
+  const formikProps = useFormikContext<IParcel>();
+
+  // lookup codes that will be used by subforms
+  const { getOptionsByType } = useCodeLookups();
+  const agencies = getOptionsByType(API.AGENCY_CODE_SET_NAME);
+  const classifications = getOptionsByType(API.PROPERTY_CLASSIFICATION_CODE_SET_NAME);
+
+  const render = (): React.ReactNode => {
+    switch (stepper.current) {
+      case 0:
+        return (
+          <div className="parcel-identification">
+            <ParcelIdentificationForm
+              agencies={agencies}
+              classifications={classifications}
+              handleGeocoderChanges={handleGeocoderChanges}
+              setMovingPinNameSpace={setMovingPinNameSpace}
+              handlePidChange={handlePidChange}
+              handlePinChange={handlePinChange}
+            />
+          </div>
+        );
+      case 1:
+        return (
+          <div className="parcel-usage">
+            <LandUsageForm classifications={classifications} nameSpace="data" {...formikProps} />
+          </div>
+        );
+      case 2:
+        return <LandValuationForm />;
+      case 3:
+        return <LandReviewPage classifications={classifications} agencies={agencies} />;
+    }
+  };
+  return (
+    <FormContentWrapper>
```

Comment:
this ends up looking a lot like the FormContentWrapper of the building form, may be some potential here to create some shared components to reduce duplication.